src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/NewAzureADAppCredentialCommand.cs

```diff
@@ -72,19 +73,16 @@ public override void ExecuteCmdlet()
                     ObjectId = ActiveDirectoryClient.GetObjectIdFromApplicationId(ApplicationId);
                 }
 
-#pragma warning disable 0618
-                if (!string.IsNullOrEmpty(Password))
-#pragma warning restore 0618
+                if (Password != null && Password.Length > 0)
```

Comment:
Nit: use if (!string.IsNullOrEmpty(Password)) as used previously

Apply to the other uses in this PR as well.

Reply:
since Password is a type of securestring,we cant use string.IsNullOrEmpty on that

---
In reply to: [148155448](https://github.com/Azure/azure-powershell/pull/4664#discussion_r148155448) [](ancestors = 148155448)

Reply:
Ah right, good catch.